package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "electricitymap-contrib",
+  "license": "MIT",
+  "devDependencies": {
+    "prettier": "^2.6.2"
+  },
+  "scripts": {
+    "prettier-fix": "prettier --write .",
+    "prettier-check": "prettier --check ."
+  }
```

Comment:
How about using `npx prettier` here, then we don't need dependencies at all at root? :)

Reply:
Yeah I was conflicted about this, but I opted for this implementation so use the same version locally and in the CI. It's possible to run npx with a version specifier but it might get harder to maintain long term.

It's an easy change though so it is up to you and the team.